tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -3052,6 +3058,26 @@ public void handlePath(String xpath) {
                         replace(fullPath, newPath, value, "Upgrade to " + TARGET_STATUS.name());
                     }
                 });
+
+        fixList.add(
+                'Z',
+                "Zero lateral: convert inheritance marker to specific value if inheritance would be lateral/problematic",
+                new CLDRFilter() {
+                    @Override
+                    public void handlePath(String xpath) {
+                        String value = cldrFileToFilter.getStringValue(xpath);
+                        if (!CldrUtility.INHERITANCE_MARKER.equals(value)) {
+                            return;
+                        }
+                        String newValue =
+                                VoteResolver.reviseInheritanceAsNeeded(xpath, value, getResolved());
+                        if (value.equals(newValue)) {
+                            return;
+                        }
+                        String fullXPath = cldrFileToFilter.getFullXPath(xpath);
+                        replace(fullXPath, fullXPath, newValue);
+                    }
+                });
```

Comment:
This new "-fZ" is results in more "hardening" than "-fV", mainly, I think, because "-fZ" doesn't distinguish between level one locales and other locales -- and also, because "-fZ" treats constructed values as lateral inheritance, while "-fV doesn't

This PR re-enables -fV, and newly enables -fZ. One or the other may be useful for revising the production data, comparing the two is instructive.